dashboard/static/css/layout.css

```diff
@@ -7,9 +7,31 @@ body {
     margin: 0 0 100px;
 }
 
+main {
+    margin-top: 20px;
+    margin-bottom: 20px;
+}
+
+a {
+    color: #00549E;
+}
+
+form {
+    margin: 20px 0;
+}
```

Comment:
Of course, I wrote this line. (And the `margin-top` and `margin-bottom` referenced in another comment.) Still, probably a good improvement to make. :smile: